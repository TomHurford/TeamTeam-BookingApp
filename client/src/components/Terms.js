import React from "react";
import "../styles/Terms.css";
//A component for Contact page to be added

function Terms() {
  return (
    <div className="page-container">
      <div className="underlay"></div>
      <h1 className="title">Terms And Conditions</h1>

      <center>
        <p>
          Welcome to our ticket selling platform. By using our platform, you
          agree to be bound by the following terms and conditions:
        </p>
        <p>
          1. User Conduct: You agree to use our platform only for lawful
          purposes and in accordance with these terms and conditions. You agree
          not to use our platform for any illegal or unauthorized purpose.
        </p>
        <p>
          2. Account Registration: You must create an account to use our
          platform. You agree to provide accurate and complete information when
          creating your account and to keep your login credentials confidential.
          You are responsible for all activity that occurs under your account.
        </p>
        <p>
          3. Event Creation and Ticket Sales: Society organizers are responsible
          for creating and managing their events on our platform. You agree to
          provide accurate and complete information when creating an event and
          to comply with all applicable laws and regulations. You are
          responsible for setting ticket prices, managing ticket inventory, and
          processing refunds.
        </p>
        <p>
          4. Payment Processing: Our platform uses third-party payment
          processors to process transactions. You agree to pay all fees
          associated with your use of our platform and to provide accurate and
          complete payment information. We reserve the right to suspend or
          terminate your account if payment is not received or is disputed.
        </p>
        <p>
          5. Intellectual Property: Our platform and all content, features, and
          functionality are owned by us or our licensors and are protected by
          intellectual property laws. You agree not to copy, modify, distribute,
          or reproduce any content from our platform without our prior written
          consent.
        </p>
        <p>
          6. Limitation of Liability: Our platform is provided on an "as is"
          basis and we make no representations or warranties regarding its
          operation or availability. We are not responsible for any damages,
          including but not limited to direct, indirect, incidental, or
          consequential damages, arising out of your use of our platform.
        </p>
        <p>
          7. Termination: We reserve the right to suspend or terminate your
          account at any time for any reason, without notice. You may also
          terminate your account at any time by contacting us.
        </p>

        <p>
          By using our platform, you agree to be bound by these terms and
          conditions. If you have any questions or concerns about these terms
          and conditions, please contact us at [insert contact information].
        </p>
      </center>
    </div>
  );
}
export default Terms;
