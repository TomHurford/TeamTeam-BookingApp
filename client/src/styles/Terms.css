.title{
    margin-top: 60px;
    text-align:left;
}

.data{
    text-align:left;
    align-self: right;

}
