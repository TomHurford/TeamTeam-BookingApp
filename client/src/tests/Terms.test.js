import React from 'react';
import renderer from 'react-test-renderer';
import Terms from '../components/Terms';

test('Terms snapshot test', () => {
  const component = renderer.create(<Terms />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});