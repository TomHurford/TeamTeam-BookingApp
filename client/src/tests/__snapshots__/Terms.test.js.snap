// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Terms snapshot test 1`] = `
<div
  className="page-container"
>
  <div
    className="underlay"
  />
  <h1
    className="title"
  >
    Terms And Conditions
  </h1>
  <p
    className="title"
  >
    Agreement to our legal terms
  </p>
  <p
    className="data"
  >
    We operate the website WEBSITE, as well as any other
    <br />
    related productes and services that refer or link to these legal terms (the 'LEGAL TERMS')(collectivly, the 'SERVICES')
    <br />
    <br />
    These Legal Terms constitute a legally binding agreement made between you, whether personally or behalf
    <br />
    of an entity("you"), and ticketopia, conserning your access to and use of the Services. You agree that by accessing the services, you 
    <br />
     have read, unstood, and agreed to be bound by all these Legal Terms
  </p>
</div>
`;
